Stack/Opc.Ua.Bindings.Https/Stack/Https/HttpsTransportListener.cs

```diff
@@ -318,16 +318,17 @@ public async Task SendAsync(HttpContext context)
                     input.RequestHeader = new RequestHeader();
                 }
 
-                if (NodeId.IsNull(input.RequestHeader.AuthenticationToken) && input.TypeId != DataTypeIds.CreateSessionRequest)
+                if (NodeId.IsNull(input.RequestHeader.AuthenticationToken) &&
+                    input.TypeId != DataTypeIds.CreateSessionRequest)
                 {
-                    if (context.Request.Headers.ContainsKey("Authorization"))
+                    if (context.Request.Headers.ContainsKey(kAuthorizationKey))
                     {
-                        foreach (string value in context.Request.Headers["Authorization"])
+                        foreach (string value in context.Request.Headers[kAuthorizationKey])
                         {
-                            if (value.StartsWith("Bearer", StringComparison.OrdinalIgnoreCase))
+                            if (value.StartsWith(kBearerKey, StringComparison.OrdinalIgnoreCase))
                             {
                                 // note: use NodeId(string, uint) to avoid the NodeId.Parse call.
-                                input.RequestHeader.AuthenticationToken = new NodeId(value.Substring("Bearer ".Length).Trim(), 0);
+                                input.RequestHeader.AuthenticationToken = new NodeId(value.Substring(kBearerKey.Length + 1).Trim(), 0);
```

Comment:
is it obvious, that value contains a substring of length kBearerKey.Length + 1?

Reply:
kBearerKey should really include the space char...